static/src/javascripts/projects/common/views/commercial/survey/survey-adblock.html

```diff
@@ -1,87 +1,113 @@
+                        </p>
+                        <p>
+                            There's no ongoing commitment, and you can stop your payments or take a holiday from ad-free at any time.
+                        </p>
                     </div>
-                    <% } %>
                 </div>
-                <% } %>
-                <% if (surveyTextUserHelp) { %>
-                <p class="survey-text__info">
-                    <%=surveyTextUserHelp%>
-                </p>
-                <% } %>
-                <% if (surveyTextMembership || surveyTextSubscriber) { %>
-                <div class="survey-links">
-                    <hr/>
-                    <% if (surveyTextMembership) { %>
-                    <p class="survey-text__info smaller">
-                        <%=surveyTextMembership%>
-                    </p>
-                    <% } %>
-                    <% if (surveyTextSubscriber) { %>
-                    <p class="survey-text__info smaller">
-                        <%=surveyTextSubscriber%>
-                    </p>
-                    <% } %>
+
+                <div class="state-4 is-hidden">
+                    <div class="survey-text__header">
+                        Thank you for your interest in an ad-free Guardian
+                    </div>
+                    <div>
+                        <p>
+                            We're assessing demand for an ad-free version of theguardian.com. Your interest has been counted
+                            and you won't be charged anything.
+                        </p>
+                        <p>
+                            Please email <a class="text-link" href="mailto:userhelp@theguardian.com">userhelp@theguardian.com</a> if you want to
+                            find out more.
+                        </p>
+                        <p>
+                            Help us improve your experience by doing our <a class="text-link" href="https://www.surveymonkey.net/r/Preview/?sm=M61PWayc8w23BetSmIe6mx6_2BhjqfmIRoVDQlyXtgbQJbKRAvwsLP8EnLFPrQHZvY" target="_blank">1 minute survey</a>.
+                        </p>
+                        <p>
+                            Thank you.
+                        </p>
+                    </div>
+
+                </div>
+
+            </div>
+        </div>
+
+        <div class="survey-buttons">
+            <div class="survey-button-rounded__cta noads state-1 state-2" data-link-name="<%=adFreeDataLink%>"><%=adFreeButtonText%><%=arrowWhiteRight%></div>
+            <div class="survey-button-rounded__cta cta-whitelist state-1" data-link-name="<%=whitelistDataLink%>"><%=whitelistText%><%=arrowWhiteRight%></div>
+            <div class="survey-button-rounded__cta readon state-4 is-hidden" data-link-name="adblock paymentdone continuetoarticle">Continue to article<%=arrowWhiteRight%></div>
+        </div>
+
+        <div class="howto-unblock state-2 is-hidden">
+            <hr />
```

Comment:
I'm not quite sure this is a semantic use of the `<hr />` element:

[MDN](https://developer.mozilla.org/en/docs/Web/HTML/Element/hr) says:
> The HTML `<hr>` element represents a thematic break between paragraph-level elements (for example, a change of scene in a story, or a shift of topic with a section)

If its just for styling, use `border-top` etc